docs/alerting/email-destination.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Destination type - Email
```

Comment:
Email Destination or Email Destination Type

Reply:
Either works. Having the "Type" in the is not a must. But I'll make it consistent.